src/parseInputs.js

```diff
@@ -93,5 +98,17 @@ export async function parseInputs(cliInputs) {
       }
     }
   }
-  return { wasmEngine, component, input, output, enableExperimentalHighResolutionTimeMethods };
+
+  if (!wasmEngine) {
+    wasmEngine = wasmEngineBase;
+    if (component) {
+      wasmEngine += "-component";
+    }
+    if (enableWeval) {
+      wasmEngine += "-weval.wasm";
+    } else {
+      wasmEngine += "-normal.wasm";
```

Comment:
These asset names are also hard-coded in https://github.com/fastly/js-compute-runtime/blob/main/.github/workflows/main.yml#L119, which I think would need to be updated too.